fe/fe-core/src/main/java/com/starrocks/common/util/KafkaUtil.java

```diff
@@ -214,7 +214,8 @@ private PProxyResult sendProxyRequest(PProxyRequest request) throws UserExceptio
                     try {
                         result = future.get(Config.routine_load_kafka_timeout_second, TimeUnit.SECONDS);
                     } catch (Exception e) {
-                        LOG.warn("failed to send proxy request to " + address + " err " + e.getMessage());
+                        LOG.warn("Error occurred during interaction between BE and Kafka broker. BE address " 
```

Comment:
I think it's useless to tell users "interaction between BE and Kafka broker", we can directly tell users "Error occurred  when getting Kafka partition meta, partition offsets, data ... . The correspoing BE address is ${address}.".
And the BE address is used to locate the problem.

And, extract the error message template and put it into error_code.h/cpp